test/view.test.ts

```diff
@@ -0,0 +1,45 @@
+import { beforeEach, describe, expect, it } from "./suite.ts";
+
+import { seedrandom } from "../src/seedrandom.ts";
+import { createGraph, normal, GraphView, createView } from "../mod.ts";
+
+describe("view", () => {
+  let view: GraphView;
+  let seed = seedrandom("normal.test.ts");
```

Comment:
I usually use the name of the test as the seed. Totally optional, but probably good to keep the convention
```suggestion
  let seed = seedrandom("view.test.ts");
```